backupper-gui/src/main/java/pl/gda/pg/eti/autyzm/Controller/RestoreController.java

```diff
@@ -38,11 +42,15 @@
     private ObservableList<JadbDevice> devices = FXCollections.observableArrayList();
     private ObservableList<DeviceCopy> copies = FXCollections.observableArrayList();
 
+
     @FXML
     public void initialize() {
         setDataBindings();
         setColumnWidth();
 
+        //mock for tests
+        copies.add(new DeviceCopy("proba-restore", LocalDate.now()));
```

Comment:
We should probably remove this line, I think it would not work if somebody doesn't have "proba-restore" backup in his/her backups.